src/main.js

```diff
@@ -2,17 +2,22 @@
 var posterImage = document.querySelector('.poster-img');
 var posterTitle = document.querySelector('.poster-title');
 var posterQuote = document.querySelector('.poster-quote');
-var showRandomButton = document.querySelector('.show-random');
+var savedPosterLocation = document.querySelector('.saved-posters-grid');
+
 var userPosterImage = document.querySelector('#poster-image-url');
 var userPosterTitle = document.querySelector('#poster-title');
 var userPosterQuote = document.querySelector('#poster-quote');
 var makePoster = document.querySelector('.make-poster');
+//buttons
+var showRandomButton = document.querySelector('.show-random');
 var showFormButton = document.querySelector('.show-form');
 var showSavedPostersButton = document.querySelector('.show-saved');
 var backToMainButton = document.querySelector('.back-to-main');
 var takeMeBackButton = document.querySelector('.show-main');
+var savePosterButton = document.querySelector('.save-poster');
+
 var savedPosters = [];
-var currentPoster;
+var currentPoster = new Poster();
```

Comment:
Play with initializing this `currentPoster` variable declaration with an initial assignment and without an initial assignment. What are the benefits of assigning it as an instantiation of `Poster`  on the declaration as opposed to assigning the instantiation only when it is needed?